### DISCUSSION TITLE
Question: how to reference another sheet in formula?

### DISCUSSION THREAD

**Discussion by Gitleptune** [andmarti1424/sc-im#717] [Q&A]
I have a file with 2 sheets, one that does formulas and the other has data, I would like to have a cell in Sheet1 that averages all of the A column in Sheet2.
In Google Sheets the syntax for referencing a cell from a different sheet is `='sheet name'!A1`.
Is there a way to do this in sc-im? I looked through :h and the issues but i couldn't find anything :/

**Comment by andmarti1424** [comment]
Please take a look at .sc test files inside tests folder.
